src/Mono.Android/Properties/AssemblyInfo.cs.in

```diff
@@ -9,6 +9,7 @@
 
 using System.Reflection;
 using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
```

Comment:
Should this file also include a [`TargetPlatfomAttribute`](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.versioning.targetplatformattribute?view=net-6.0)?

```csharp
[assembly: TargetPlatformAttribute ("android@API_LEVEL@")]
```

Reply:
I made this PR also fix: https://github.com/xamarin/xamarin-android/issues/6501